packages/tocco-ui/src/Select/DropdownIndicator.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 
 import Button from '../Button'
 
-const DropdownIndicator = () =>
-  <span>
-    <Button
-      icon="chevron-down"
-      look="ball" />
-  </span>
+const DropdownIndicator = props =>
+  props.isDisabled
```

Comment:
Would this also work?

```
const DropdownIndicator = props =>
   props.isDisabed === false && <Button ..../>
```

a tenary that returns null in case of true seams a bit odd.

Reply:
see 112dbc4cce34971f5044ca0e8b94d063de1726aa